### ISSUE TITLE
Crash when dragging an app icon with dynamic content (clock)

### ISSUE THREAD

**Issue by l3u** [otakuhqz/Omega#38]
As of 0.6.2 (installed via the kindly provded F-Droid package), dynamic icons are shown, e. g. the clock app's icon now shows the current time. But: As soon as one holds the icon to e. g. move it, Omega crashes. Happens each time, and both for trying to drag around an icon on the desktop or from the drawer.

* Device: Nokia 4.2
* Android version: 11
* Launcher version: 0.6.2

Here's the crash log:
```
Build version: 0.6.2 
Build date: 1981-01-01 01:01:02 
Current date: 2021-01-14 12:26:11 
Device: HMD Global Nokia 4.2 
 
Stack trace:  
java.lang.IllegalStateException: unwrapped AdaptiveIconDrawable for ApplicationInfo{6a4b3ef com.google.android.deskclock}
	at com.saggitt.omega.icons.CustomIconProvider.assertNotAdaptiveIconDrawable(CustomIconProvider.kt:61)
	at com.saggitt.omega.icons.CustomIconProvider.getIcon(CustomIconProvider.kt:43)
	at com.android.launcher3.icons.IconCache.getFullResIcon(IconCache.java:242)
	at com.android.launcher3.Utilities.getFullDrawable(Utilities.java:576)
	at com.android.launcher3.dragndrop.DragView$2.run(DragView.java:219)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:214)
	at android.os.HandlerThread.run(HandlerThread.java:67)
```

**Comment by otakuhqz** [comment]
The crash for dynamic icons has been already fixed in the current development version, however, I just noted that the app crash when dragging icons from the apps suggestions. So it needs to be revised.

**Comment by l3u** [comment]
Just one question (I didn't want to open a new issue for that): I installed 0.6.2 from the additional F-Droid repo sources, you talk about 0.6.3 – and the latest tag here is 0.5.0? Also, the repo seems to contain a package named "Omega-NEO" … this is a bit confusing … where do the additional tags come from and what's Omega-NEO?

**Comment by otakuhqz** [comment]
Add the below repo to the f-droid app. we are setting up two repos: one taking the APK from releases (last release v0.5.0 build 40, package: com.saggitt.omega) and the other one the current debug version (last version v0.6.3 build 105, package com.saggitt.omega.dev ).
[https://github.com/rafaelvenancio98/fdroid/tree/master/fdroid/repo](https://github.com/rafaelvenancio98/fdroid/tree/master/fdroid/repo)

If you want to have the most updated debug versions, you can also get it from Github actions (click in one job and navigate to the bottom, then download the zip file).
[https://github.com/otakuhqz/Omega/actions](https://github.com/otakuhqz/Omega/actions)

**Comment by l3u** [comment]
I just added it and get "Error getting F-Droid index file → error in opening zip file" … :-(

**Comment by l3u** [comment]
https://raw.githubusercontent.com/rafaelvenancio98/fdroid/master/fdroid/repo (the link from issue #9) seems to work thoug …

**Comment by otakuhqz** [comment]
You are right, the correct repo is [https://raw.githubusercontent.com/rafaelvenancio98/fdroid/master/fdroid/repo](https://raw.githubusercontent.com/rafaelvenancio98/fdroid/master/fdroid/repo)

**Comment by l3u** [comment]
I just installed 0.7.1 via the Github package (alas, it's still not installable via the F-Droid repo), and draging the clock on the desktop still crashes the app. So it doesn't seem to be fixed … :-(

**Comment by otakuhqz** [comment]
I will check it again

**Comment by otakuhqz** [comment]
fixed on v0.8
